docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -756,4 +801,57 @@ subject: |
 content: |
   The case number is usually on the top right of your court forms. Below is an image where you can find this info.
 
-  ${ image_case_number.show(width="100%") }
\ No newline at end of file
+  ${ image_case_number.show(width="100%") }
+  
+---
+sets:
+  - users[i].has_attorney
+id: user has attorney
+question: |
+  Do you have a lawyer?
+subquestion: |
+fields:
+  - no label: users[i].has_attorney
+    input type: radio
+    choices:
+      - Yes: True
+      - No: False
+---
+sets:
+  - x.has_attorney
+id: person has attorney
+generic object: ALIndividual
+question: |
+  Does ${ x } have a lawyer?
+subquestion: |
+fields:
+  - no label: x.has_attorney
+    input type: radio
+    choices:
+      - Yes: True
+      - No: False
+---
+sets:
+  - x[i].name.first
+  - x[i].name.last
+  - x[i].name.middle
+  - x[i].name.suffix
+  - x[i].bar_number
+id: lawyer name and bar
+generic object: ALPeopleList
+question: |
+  What is the lawyer's name and bar number?
+subquestion:
+  ${ collapse_template(help_user_find_bar_number_template) }
+fields:
+  - code: |
+      x[i].name_fields()
+  - Bar number: x[i].bar_number
```

Comment:
Should we specify whether to include the P? My preference is that they should include the P. I think that just feels more intuitive.

Reply:
Good point, and agreed. I suppose it really is part of the bar number, and if there's ever a form that already includes the 'P' it can just be stripped from the user's input prior to being put on the form.